Description: Fix from upstream for leap year handling
Last-Updated: 2024-03-12
Forwarded: not-needed

--- a/slsh/lib/timestamp.sl
+++ b/slsh/lib/timestamp.sl
@@ -191,7 +191,7 @@
    day--;			       %  offset from 0
    % Compute the cumulative number of days, accounting for a leap year
    day += Cumulative_Days_Per_Month[month];
-   if ((month > 2)
+   if ((month >= 2)  % month runs 0 .. 11
        && (0 == (year mod 4))
        && ((year mod 100) || (0 == (year mod 400))))
      day++;
